open-sphere-base/overlay/src/main/java/io/opensphere/overlay/CursorPositionPopupManager.java

```diff
@@ -115,11 +120,88 @@ public void eventOccurred(InputEvent event)
                     dialog.setLocationRelativeTo(dialog.getParent());
                     dialog.pack();
                     dialog.setVisible(true);
+
                 });
             }
         }
     };
 
+    /**
+     * The key listener that will copy the coordinates to the clipboard in
+     * preferred format.
+     */
+    private final DiscreteEventAdapter myAltListener = new DiscreteEventAdapter("Cursor Position", "Display Cursor Position",
+            "Show a popup with the current mouse cursor position")
+    {
+
+        private String myCoordLabel;
+
+        @Override
+        public void eventOccurred(InputEvent event)
+        {
+            if (myLocation != null)
+            {
+                EventQueueUtilities.invokeLater(() ->

```

Comment:
Does this need to be on the event queue?

Reply:
after testing it appears it doesn't need to be for the copy; unsure about the original code for the popup menu